src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java

```diff
@@ -290,30 +289,44 @@ public String replaceDocRootDir(String htmlstr) {
      * @param dl the content to which the method information will be added
      */
     private void addMethodInfo(ExecutableElement method, Content dl) {
```

Comment:
It's not anything you've changed, and not part of the problem you're fixing, but this doesn't feel like it belongs on `HtmlDocletWriter`.  But that's another story of different architectural Technical Debt.